src/config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "monday-trigger",
+  "type": "trigger",
+  "execProgram": "node",
+  "main": "app.js",
+  "exposeRoute": true,
+  "imgUrl": "monday.png",
+  "version": "1.0.0",
+  "description": "Handle Monday Webhooks",
+  "methods": [
+    {
+      "name": "STATUS_UPDATE_WEBHOOK",
+      "route": "post /webhook/monday/alert",
+      "viewName": "Monday alert webhook",
```

Comment:
@ilanyaniv I think the name is incorrect, this is not an `alert` webhook but a `board change`. 
Beside, please change the route to fit its purpose, so if it is a `board change` I would change the route to `/webhook/monday/board`